.github/workflows/tag-and-release.yml

```diff
@@ -46,12 +48,38 @@ jobs:
       - name: Login to registry1
         run: uds run registry-login --set REGISTRY=registry1.dso.mil --set REGISTRY_USERNAME=${{ secrets.IRON_BANK_ROBOT_USERNAME }} --set REGISTRY_PASSWORD=${{ secrets.IRON_BANK_ROBOT_PASSWORD }} --set REGISTRY_RETRY_INTERVAL=90
 
+      - name: Create Standard Package and Bundle
+        if: ${{ matrix.type == 'standard-package' }}
+        run: |
+          set -e
+          ZARF_ARCHITECTURE=${{ matrix.arch }} uds run -f tasks/create.yaml standard-package --no-progress --set FLAVOR=${FLAVOR}
+          ZARF_ARCHITECTURE=${{ matrix.arch }} uds run -f tasks/create.yaml k3d-standard-bundle --no-progress
+
+      - name: Create Istio Package and Bundle
+        if: ${{ matrix.type == 'istio-package'}}
+        run: |
+          ZARF_ARCHITECTURE=${{ matrix.arch }} uds run -f tasks/create.yaml istio-package --no-progress --set FLAVOR=${FLAVOR}
+          ZARF_ARCHITECTURE=${{ matrix.arch }} uds run -f tasks/create.yaml k3d-istio-bundle --no-progress
+
+      - name: Deploy Standard Package via Bundle
+        if: ${{ matrix.type == 'standard-package' && matrix.arch == 'amd64' }}
+        run: uds run -f tasks/deploy.yaml k3d-standard-bundle --no-pogress --set uds_arch=${{ matrix.arch }}
+
+      - name: Deploy Istio Packages via Bundle
+        if: ${{ matrix.type == 'istio-package' && matrix.arch == 'amd64' }}
+        run: uds run -f tasks/deploy.yaml k3d-istio-bundle --no-pogress --set uds_arch=${{ matrix.arch }}
```

Comment:
Commenting here, but this applies to all the items here. I think if we changed the matrix `type`s to `standard` and `istio` we could DRY this up:
```
      - name: Deploy ${{ matrix.type }} Package via Bundle
        if: ${{ matrix.arch == 'amd64' }}
        run: uds run -f tasks/deploy.yaml k3d-${{ matrix.type }}-bundle --no-pogress --set uds_arch=${{ matrix.arch }}
```

Something like this _should_ handle both standard/istio and work for create, deploy, and publish I think?

Reply:
mmm, nice point of refactor. i like it.